From 5625949232f351eadc4437fc4ba6fb430ff06435 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Mon, 16 Jan 2017 12:14:00 -0800
Subject: [PATCH 49/74] Use BOARD_HAVE_QCOM_FM flag

* Prevent parsing any subdirectory files unless BOARD_HAVE_QCOM_FM is
  enabled.

Change-Id: I7f4995b882bac3a6380026eaab2dedd57b5bb5b8
---
 Android.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 8c8d2c8..65587c8 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,10 +1,11 @@
+ifeq ($(BOARD_HAVE_QCOM_FM),true)
+
 LOCAL_PATH:= $(call my-dir)
 LOCAL_DIR_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_MODULE_TAGS := optional
 
-
 LOCAL_SRC_FILES := $(call all-java-files-under, qcom/fmradio)
 LOCAL_JNI_SHARED_LIBRARIES := libqcomfm_jni
 
@@ -41,3 +42,5 @@ include $(LOCAL_PATH)/helium/Android.mk
 
 LOCAL_PATH := $(LOCAL_DIR_PATH)
 include $(LOCAL_PATH)/libfm_jni/Android.mk
+
+endif # BOARD_HAVE_QCOM_FM
-- 
2.17.1

